### PR TITLE
Add support for string concatenation in keys

### PR DESCRIPTION
**Pull request by gillyb**

### CODE REVIEW

File: hoconjs.js [hunk 1]
```diff
@@ -83,6 +83,11 @@ function parseHocon(text) {
         switch (c) {
           case ' ':
             {
+              if (currentKey !== '' && !isReadingValue) {
+                currentKey += c;
+                console.log('adding to key : ' + c);
```
Comment (ghost) [inline hoconjs.js]: Remove :)

File: tests.js [hunk 2]
```diff
@@ -208,4 +208,16 @@ QUnit.test('String concatenation on array values without comma', function(
   assert.equal(obj.a[0], '1 2 3 4');
 });
 
+QUnit.test('String concatenation in field keys', function(assert) {
+  var obj = parseHocon(`{
+    a: 42,
+    a b : 3
+  }`);
+  console.log(JSON.stringify(obj));
```
Comment (ghost) [inline tests.js]: Remove

File: tests.js [hunk 3]
```diff
@@ -208,4 +208,16 @@ QUnit.test('String concatenation on array values without comma', function(
   assert.equal(obj.a[0], '1 2 3 4');
 });
 
+QUnit.test('String concatenation in field keys', function(assert) {
+  var obj = parseHocon(`{
+    a: 42,
+    a b : 3
+  }`);
```
Comment (ghost) [inline tests.js]: Sadly our phantomJS build doesn't support ES6, either convert to ES5 or add a build step that does it in Travis

**Review by yellowblood — CHANGES_REQUESTED** [review]

File: hoconjs.js (hunk 1 above)
Comment (gillyb) [reply hoconjs.js]: Oh shit! didn't mean to leave these of course!

File: tests.js (hunk 3 above)
Comment (gillyb) [reply tests.js]: oh, bummer.. ok, might add the build step in the future, for now i'll fix the code.

**Comment by gillyb** [comment]
@yellowblood I updated the code, just take a look now, and merge if ok

**Review by yellowblood — APPROVED** [review]

**Pull request merged**